### PR TITLE
Do not set gammaInput=true, and ease back the ambient lighting a bit.

### PR DESCRIPTION
**Pull request by jsantell**
Before:

screenshot from 2018-11-13 08-43-51
screenshot from 2018-11-13 08-43-37

After:

screenshot from 2018-11-13 08-43-02

screenshot from 2018-11-13 08-42-49

### CODE REVIEW

**Comment by mrdoob** [comment]
You may have to set `texture.encoding = THREE.GammaEncoding` on the skysphere.

**Comment by jsantell** [comment]
Setting GammaEncoding on just the equirect makes it obvious that it's a bit blown out still in the environment map.

screenshot from 2018-11-13 10-50-16

Applying it to all (LDR only for now) textures that come in (so that it's marked as sRGB in both sky/env) looks pretty good now:

screenshot from 2018-11-13 10-50-34

File: src/three-components/ModelScene.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ export default class ModelScene extends Scene {
 
     this.model = new Model();
     this.shadow = new StaticShadow();
-    this.light = new AmbientLight(0xffffff, 1.5);
+    this.light = new AmbientLight(0xffffff, 1);
```
Comment (cdata) [inline src/three-components/ModelScene.js]: 👍

File: src/three-components/TextureUtils.js [hunk 2]
```diff
@@ -43,9 +43,11 @@ export default class TextureManager extends EventDispatcher {
    * @param {string} url
    * @return {Promise<THREE.Texture>}
    */
-  load(url) {
-    return new Promise(
+  async load(url) {
+    const texture = await new Promise(
         (resolve, reject) => loader.load(url, resolve, undefined, reject));
+    texture.encoding = GammaEncoding;
```
Comment (cdata) [inline src/three-components/TextureUtils.js]: 👍

File: src/three-components/Renderer.js [hunk 3]
```diff
@@ -54,7 +54,6 @@ export default class Renderer extends EventDispatcher {
         new WebGLRenderer({canvas: this.canvas, context: this.context});
     this.renderer.autoClear = false;
     this.renderer.setPixelRatio(DPR);
-    this.renderer.gammaInput = true;
```
Comment (cdata) [inline src/three-components/Renderer.js]: 👍

**Pull request merged**